php-practice_1.php

```diff
@@ -0,0 +1,181 @@
+<?php
+// Q1 変数と文字列
+$name = '安藤';
+echo '私の名前は「' . $name . '」です';
+
+
+// Q2 四則演算
+$num = (5 * 4);
+echo $num;
+
+$num /= 2;
+echo $num;
+
+
+// Q3 日付操作
+$time = date("Y年m月d日 H時i分s秒");
+echo "現在時刻は、" . $time . "です。";
+
+
+// Q4 条件分岐-1 if文
+$device = 'mac';
+
+if ($device === 'windows') {
+    echo '使用OSは、windowsです。';
+} else {
+    if ($device === 'mac') {
+    echo '使用OSは、macです。';
+} else {
+    echo 'どちらでもありません。';
+}
+}
+
+
+// Q5 条件分岐-2 三項演算子
+$age = 18;
+$message = ($age > 18) ? '未成年です。' : '成人です。';
+echo $message;
+
+
+// Q6 配列
+$prefecture = ['神奈川県', '北海道' ,'栃木県', '千葉県'];
+echo $prefecture[2] . 'と' . $prefecture[3] . 'は関東地方の都道府県です。';
+
+
+// Q7 連想配列-1
+$ken = [
+  "東京都" => "新宿区",
+  "神奈川県" => "横浜市",
+  "千葉県" => "千葉市",
+  "埼玉県" => "さいたま市",
+  "栃木県" => "宇都宮市",
+  "群馬県" => "前橋市",
+  "茨城県" => "水戸市"
+];
+
+foreach ($ken as $key => $name) {
```

Comment:
[info]
変数名は格納されている値が読み取れる名前がいいと思います！
例えば...
```suggestion
foreach ($ken as $prefecture => $city) {
```

Reply:
承知いたしました。
確かにprefecturesにした方が、修正や変更があった時に何を指しているのか分かりやすいです。

今回はアドバイスいただいた通りに変更いたしました。

foreach ($ken as $prefectures => $city) {
  echo $city . "\n";
}